fraqbot/tests/test_randomemoji.py

```diff
@@ -0,0 +1,52 @@
+import os
+import re
+import sys
+import threading
+
+from Legobot.Lego import Lego
+from mock import patch
+
+
+LOCAL_PATH = os.path.join(
+    os.path.abspath(os.path.dirname(__file__)),
+    '..',
+    'Local'
+)
+sys.path.append(LOCAL_PATH)
+
+
+from randomemoji import RandomEmoji  # noqa: E402
+
+
+LOCK = threading.Lock()
+BASEPLATE = Lego.start(None, LOCK)
+LEGO = RandomEmoji(BASEPLATE, LOCK)
+
+
+def test_listening_for():
+    # !emoji
+    assert LEGO.listening_for({'text': 'emoji'}) is False
+    assert LEGO.listening_for({'text': '!emoji moin'}) is True
+    assert LEGO.listening_for({'text': '!emoji 12'}) is True
+    assert LEGO.listening_for({'text': '!Emoji'}) is False
+
+
+def test_get_emoji():
```

Comment:
This test should be failing (the test job didn't run when you submitted, because I didn't have it set to run on external PRs, just internal pushes), since the Lego is started without config.

It would be better to mock `utils.call_slack_api` and return a mocked emoji value.